### DISCUSSION TITLE
Point Release 2.0.1

### DISCUSSION THREAD

**Discussion by grempe** [ryan-mars/ulid-workers#28] [General]
Hi, with the recent series of commits I think I've added what I would intend to add for now.

If there's nothing else pending perhaps you can drop a new point release to NPM? I think it could be a semver patch release as I don't think any new API changes have crept in since the 2.0.0 release.

Cheers.

**Comment by ryan-mars** [comment]
Good idea.

**Comment by ryan-mars** [comment]
2.1.0 was released yesterday
